lib/Onyx.ts

```diff
@@ -594,6 +594,42 @@ function clear(keysToPreserve: OnyxKey[] = []): Promise<void> {
         .then(() => undefined);
 }
 
+function updateSnapshots(data: OnyxUpdate[]) {
+    const snapshotCollectionKey = OnyxUtils.getSnapshotKey();
+    if (!snapshotCollectionKey) return;
```

Comment:
WE have a rule in NewDot to always add curly braces, I'm surprised it's not in the Onyx too 🤔 

```suggestion
    if (!snapshotCollectionKey) {
      return;
    }
```